File: net472/webapi/BuildableWebApi/BuildableWebApi/DataAccess/SqlDbAccess.cs
using System;
using System.Collections.Generic;
using System.Configuration;
using System.Data.SqlClient;
using System.Linq;
using System.Web;

namespace BuildableWebApi.DataAccess
{
    public class SqlDbAccess
    {
        public string GetData()
        {
            var connectionString = ConfigurationManager.ConnectionStrings["SqlDbAccess"].ConnectionString;
            var dataResult = new List<string>();
            using (var sqlConnection = new SqlConnection(connectionString))
            {
                using (var sqlCommand = new SqlCommand("SELECT * FROM Table", sqlConnection))
                {
                    sqlConnection.Open();


                    var sqlDataReader = sqlCommand.ExecuteReader();
                    while (sqlDataReader.Read())
                    {
                        dataResult.Add(sqlDataReader["ColumnName"].ToString());
                    }

                    sqlConnection.Close();
                }
            }
            return String.Join(",", dataResult);
        }
    }
}

File: net472/VBBuildableWebApi/VBWebApi/SqlDbAccess.vb
Imports System
Imports System.Collections.Generic
Imports System.Configuration
Imports System.Data.SqlClient
Imports System.Linq
Imports System.Web


Public Class SqlDbAccess

    public Function GetData As String
        dim connectionString = ConfigurationManager.ConnectionStrings("SqlDbAccess").ConnectionString
        dim dataResult As List(Of String) = New List(Of String)()
        Using sqlConnection As SqlConnection = new SqlConnection(connectionString)
            sqlConnection.Open()
            using sqlCommand = New SqlCommand("SELECT * FROM Table", sqlConnection)
                dim sqlDataReader = sqlCommand.ExecuteReader()
                While sqlDataReader.Read()
                    dataResult.Add(sqlDataReader("ColumnName").ToString())
                End While
                sqlConnection.Close()
            end Using 
        End Using

        return String.Join(",", dataResult)
    End Function
End Class
